anthem/lyrics/loaders.py

```diff
@@ -2,6 +2,8 @@
 # Copyright 2016-2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
+from past.builtins import basestring
```

Comment:
Shouldn't the lib be added in setup.py?

Reply:
Seems that we need future to have past !?
Added to setup.py.

Reply:
Yes! :sweat_smile: